ephios/modellogging/models.py

```diff
@@ -0,0 +1,252 @@
+import itertools
+import threading
+from datetime import date, datetime, time
+from json import JSONEncoder
+from typing import TYPE_CHECKING
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.db.models import QuerySet
+from django.utils.formats import localize
+from django.utils.functional import SimpleLazyObject, cached_property
+from django.utils.translation import gettext_lazy as _
+
+from ephios.modellogging.recorders import (
+    InstanceActionType,
+    M2MLogRecorder,
+    ModelFieldLogRecorder,
+    capitalize_first,
+    register_log_recorders,
+)
+
+# pylint: disable=protected-access
+
+
+class LogJSONEncoder(JSONEncoder):
```

Comment:
isn't that what we are already doing in our CustomJSONEncoder?

Reply:
Not exactly. I just changed the JSONEncoders again. 
Also, as this app is designed in a standalone way, I don't want to import ephios code.

Reply:
(although I am using the one permission function...)